components/posts/posts.php

```diff
@@ -253,6 +274,277 @@ function mm_posts_output_masonry_sizers() {
  * @param  object  $context  The global post object.
  * @param  array   $args     The instance args.
  */
+function mm_post_article( $post, $context, $args ) {
+
+	ob_start(); ?>
+
+	<article id="post-<?php the_ID( $post ); ?>" <?php post_class( 'mm-post' ); ?> itemscope itemtype="http://schema.org/BlogPosting" itemprop="blogPost" aria-label="Article">
```

Comment:
Big bonus points for you if you introduce `mm_posts_output_schema()` and allow dynamic schema definitions based on a passed arg. There are a few common schema definitions that I think you could make available in a dropdown field, maybe:

Article
BlogPosting
NewsArticle
ScholarlyArticle
SocialMediaPosting
TechArticle

From: http://schema.org/docs/full.html

Then of course a filter on the output of that function to allow other schema types like Event.